sympy/rubi/tests/test_algebraic_set2.py

```diff
@@ -0,0 +1,3566 @@
+# Maple integration test file: "1 Algebraic functions\1 Linear products\1.3 (a+b x)**m (c+d x)**n (e+f x)**p.txt"
+
+#lst:=[
+
+# Integrands of the form (e x)**m (a+b x)**n (c+d x)**p
+
+# Integrands of the form (e x)**m (a+b x)**n (c+d x)**p
+
+# Integrands of the form (e x)**m (a+b x)**1 (c+d x)**p when b c+a d=0
```

Comment:
No need to change this line, Just write what does `test_set1`, `test_set2` and `test_set3` mean? How have you divided the tests?

Write these on top line of the file as comment